src/components/Notes-Card/notescard.css

```diff
@@ -0,0 +1,130 @@
+:root {
+  --notes-red-bg: red;
+  --notes-blue-bg: blue;
+  --notes-green-bg: green;
+  --notes-yellow-bg: yellow;
```

Comment:
use hex codes, as names differs from browser to browser. Your app will not look consistent throughout different browsers